scripts/nuget.fsx

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env fsharpi
+
+open System
+open System.IO
+open System.Linq
+
+#r "System.Configuration"
+#load "../fsx/InfraLib/Misc.fs"
+#load "../fsx/InfraLib/Process.fs"
+#load "../fsx/InfraLib/Network.fs"
+open FSX.Infrastructure
+open Process
+
+let nugetDownloadUri = Uri "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"
+Network.DownloadFile nugetDownloadUri
+
+// this is a translation of doing this in unix:
+// 0.1.0-date`date +%Y%m%d-%H%M`.git-`echo $GITHUB_SHA | cut -c 1-7`
+let GetIdealNugetVersion (initialVersion: string) =
+    let dateSegment = sprintf "date%s" (DateTime.UtcNow.ToString "yyyyMMdd-hhmm")
+    let githubEnvVarNameForGitHash = "GITHUB_SHA"
+    let gitHash = Environment.GetEnvironmentVariable githubEnvVarNameForGitHash
+    if gitHash = null then
+        //TODO: in this case we should just launch a git command
+        failwithf "Environment variable %s not found, not running under GitHubActions?"
+                  githubEnvVarNameForGitHash
+    let gitHashDefaultShortLength = 7
+    let gitShortHash = gitHash.Substring(0, gitHashDefaultShortLength)
+    let gitSegment = sprintf "git-%s" gitShortHash
+    let finalVersion = sprintf "%s.0-%s.%s"
+                               initialVersion dateSegment gitSegment
+    finalVersion
+
+let PackAndMaybeUpload (packageName: string) =
+    let rootDir = DirectoryInfo(Path.Combine(__SOURCE_DIRECTORY__, ".."))
+    let nugetExe = Path.Combine(rootDir.FullName, "nuget.exe") |> FileInfo
```

Comment:
you're actually not using the downloaded nuget.exe, are you? (this is a good thing) then we can remove all the lines related to this (including the `Network.DownloadFile`)

Reply:
I didn't even notice that I wasn't using nuget.exe, I will remote it plus all the other unneeded code